client/landing/gutenboarding/components/locale-context/index.tsx

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
Would there be better, more generic place for this component than in `client/landing/gutenboarding/components/` ?

cc @akirk for thoughts.